lib/streamlit/type_util.py

```diff
@@ -187,7 +211,7 @@ def is_keras_model(obj: Any) -> bool:
     )
 
 
-def is_plotly_chart(obj: Any) -> bool:
+def is_plotly_chart(obj: Union[Figure, Collection[object]]) -> bool:
```

Comment:
Not sure if we should use `Union[Figure, Collection[object]]` instead of `object` here as with most of the other `is_` methods. Is there a specific reason why you decided to specify the type here?

Reply:
Not sure what I was thinking here. Will fix.

Reply:
Should be all good now.